mlem/core/requirements.py

```diff
@@ -425,21 +480,13 @@ def new(cls, requirements: "AnyRequirements" = None):
             return Requirements(__root__=[])
         return resolve_requirements(requirements)
 
-    def materialize_custom(self, path: str):
```

Comment:
lets keep it, just delegate to `CustomRequirement.materialize`. It will allow omit unnecessary imports of `CustomRequirement`

Reply:
done